.changeset/bright-bears-bake.md

```diff
@@ -0,0 +1,8 @@
+---
+'imagetools-core': minor
+'rollup-plugin-imagetools': minor
+'vite-imagetools': minor
+---
+
+Log messages through Vite and Rollup. Log level can be controlled in Vite and Rollup through the `logLevel` and `onwarn`
+options respectively.
```

Comment:
there should not be a line break in the changeset
```suggestion
feat: log messages through Vite and Rollup. The log level can be controlled in Vite and Rollup through the `logLevel` and `onwarn` options respectively.
```

Reply:
Seems the linebreak is needed to satisfy prettier?

Reply:
Ah, I don't think I've run into having a long message before :smile: How about we just shorten it up?
```suggestion
feat: log messages through Vite and Rollup. Allows the log level to be set with `logLevel` and `onwarn` respectively.
```